apps/system/utils/netcmd.c

```diff
@@ -606,14 +598,6 @@ const static tash_cmdlist_t net_appcmds[] = {
 #ifdef CONFIG_SYSTEM_NETDB
 	{"netdb", netdb_main, TASH_EXECMD_SYNC},
 #endif
-#ifdef CONFIG_EXTERNAL_WAKAAMA_EXAMPLE
-#ifdef CONFIG_LWM2M_CLIENT_MODE
-	{"lwm2mclient", lwm2m_client_main, TASH_EXECMD_SYNC},
-#endif
-#ifdef CONFIG_LWM2M_SERVER_MODE
-	{"lwm2mserver", lwm2m_server_main, TASH_EXECMD_SYNC},
-#endif
-#endif
```

Comment:
Dear. dr-venkman

Sorry for late response.
We're preparing next pull request including separation of example for DS SW R&D center
all wakaama codes will be reverted to remove DS SW R&D center changes

Thanks